man/veritytab.xml

```diff
@@ -60,13 +60,61 @@ This is based on crypttab(5).
 
     <variablelist class='fstab-options'>
 
+      <varlistentry>
+        <term><option>no-superblock</option></term>
+
+        <listitem><para>Use dm-verity without permanent on-disk superblock.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>format=number</option></term>
```

Comment:
like in the earlier comment, please uppercase the rhs and enclose in `<replaceable>`

Reply:
(here and everywhere else)